package message;
import "PropInfo.proto";
option optimize_for = LITE_RUNTIME; 
option java_outer_classname = "HSPropInfoData";

message PropInfoData
{
	repeated	PropInfo		manualPropInfoList	= 1;//所有手动道具信息
	repeated	PropInfo		bufferPropInfoList	= 2;//所有buffer道具信息
	repeated	PropInfo		giftBagPropInfoList = 3;//礼包道具信息
	repeated  PropInfo    cheapPropInfoList = 4;//打折道具信息
	repeated  PropInfo    lockPropInfoList  =5;//道具槽锁信息
}